streams/src/test/java/org/apache/kafka/streams/integration/NamedTopologyIntegrationTest.java

```diff
@@ -676,7 +677,7 @@ public void shouldAddToEmptyInitialTopologyRemoveResetOffsetsThenAddSameNamedTop
         }
     }
 
-    @Test
+    @Ignore
```

Comment:
I ignored the test since it failed with the processing threads, and I didn't want to fix it (now) since named topologies are probably going to be removed by the time this sees the light of day. However, it should not fail in the default configuration, so I removed the ignore